feature-libs/cart/saved-cart/components/add-to-saved-cart/add-to-saved-cart.component.html

```diff
@@ -12,6 +12,7 @@
       <span>{{ 'addToSavedCart.savedCarts' | cxTranslate }}</span>
     </button>
     <button
+      *ngIf="showSaveCartForLater(cart)"
```

Comment:
It would be better if we use an observable here. Otherwise the function will keep being recalled.